### ISSUE TITLE
Real time communication/collaboration 

### ISSUE THREAD

**Issue by sks** [trivago/gollum#95]
Are there any plans to use chat tool like gitter for this repo for collaboration?

**Comment by andygrunwald** [comment]
@sks Up to now there are no such plans. Communication via issues and PRs worked quite well so far. Any special questions from your side?

**Comment by sks** [comment]
I like the m:n mutliplexer idea and wanted to use it for my custom use case like Mqtt to websocket and things like that.

So wanted to check if there was a platform where producer/subscriber developers could collaborate so that we could reuse code.

https://github.com/sks/mqtt

**Comment by andygrunwald** [comment]
Such a platform don't exist (yet). Maybe we can open up the wiki for this? @arnecls @ppar @msiebeneicher whats your opinion?

**Comment by arnecls** [comment]
In general I think it could be a good idea. I guess especially with the 0.5.0 changes there will be a lot of questions because of the many breaking changes.
I would prefer slack though because we use it here anyway and it seems to be widely adopted.

**Comment by ppar** [comment]
I'd also prefer Slack and/or Wiki over yet-another-messaging-app.

(IRC would be fine for me too, but maybe it's a bit old-hat and not as popular)

**Comment by arnecls** [comment]
Ok, slack it is - I'll take care of this.

**Comment by arnecls** [comment]
ok, I created a slack team: gollum.slack.com.
Let's see how this works out.

**Comment by arnecls** [comment]
Ok, first obstacle - finding an easy way to request invites to a slack team.
I tried [slackin](https://github.com/rauchg/slackin), but that seems to require guest login which is only available for paid teams 🙄  ... We could use the invitation link but that seems to have limits.
Any ideas?

**Comment by arnecls** [comment]
After some internal discussion we decided to go with gitter: https://gitter.im/trivago/gollum?utm_source=share-link&utm_medium=link&utm_campaign=share-link

I also added this to the readme file.
The main reasoning behind this is that we want it to be as easy as possible for contributors to connect and ask questions. For slack it would have required a third party app plus handling invites. IRC might be alien for some developers (e.g. MacOSX has no up to date GUI client).
Gitter can be accessed via browser and has an option to login with you github account which avoids creating a new account and downloading an extra app.
So we created a gitter community. Let's see how this works out.
